lavague-core/lavague/core/base_driver.py

```diff
@@ -108,6 +108,22 @@ def get_html(self, clean: bool = True) -> str:
         """
         pass
 
+    @abstractmethod
+    def get_tabs(self) -> str:
+        """Return description of the tabs opened with the current tab being focused.
+
+        Example of output:
+        Tabs opened:
+        0 - Overview - OpenAI API
+        1 - [CURRENT] Nos destinations Train - SNCF Connect
+        """
+        pass
+
+    @abstractmethod
+    def switch_tab(self, tab_id: int) -> None:
+        """Switch to the tab with the given id"""
+        pass
+
```

Comment:
```suggestion
```
get_tabs and switch_tabs are already defined as non-abstract in the same class

Reply:
It was a merge issue, it's now gone